zip-draft-multisig-dev-fund.rst

```diff
@@ -0,0 +1,177 @@
+  
+  - Methods for measuring community sentiment MAY also include Zcash Forum user signalling limited to accounts created before a certain date. Ideally, all  ZEC holders would also be able to signal their support of specific proposals but, given current technical limitations, this may not be feasible. The outcome of the two procedures listed above, as well as any other that end up getting implemented for the same purpose, SHOULD play a central role in determining the official position of both the ECC and the  ZF on the issue described in the Motivation section of this proposal.
+
+- Question of whether the ECC should reorganize itself into a non-profit, as suggested by the ZF in their August 6, 2019 statement. [#ZFSTATEMENT]_ The current proposal neither prescribes nor excludes the option of the ECC becoming a non-profit. The authors consider the basic governance rules of the Zcash Development Fund outlined above sufficient to ensure transparency and accountability, regardless of whether the Applicant is a for-profit or a non-profit entity. According to the current proposal, funding from the Zcash Development Fund would be available to both for- and non-profit entities, thereby allowing for maximal flexibility in terms of the types of activities that can be funded.
+
+Security Considerations
+============
+
+To be completed.
+
+Trade-offs between 2-of-2, 2-of-3, and 3-of-3.
+
+Internal security and key management practices within each of the governing entities.
+
+Discussion
+============
+
+Recognized objections to this proposal include:
+
+- It is not in accordance with the current protocol specification and the initial promise of the creators of Zcash, according to which 100% of coinbase rewards will go to miners after the first halving. The main counter-argument that motivates the current proposal concerns the need to guarantee stable and sufficient funding for world-class cryptographers, engineers, and other professionals to continue contributing their time and effort to Zcash;
+
+- Objections concerning the various parameters of the Zcash Development Fund funding mechanism described above;
+
+- Objections concerning the governance rules of the Zcash Development Fund described above.
+
+References
+==========
+
+.. [#RFC2119] `Key words for use in RFCs to Indicate Requirement Levels <https://tools.ietf.org/html/rfc2119>`_
+.. [#NUPIPELINE] `The Zcash Network Upgrade Pipeline <https://electriccoin.co/blog/the-zcash-network-upgrade-pipeline/>`_
+.. [#ABOUTZF] `About the Zcash Foundation <https://www.zfnd.org/about/>`_
+.. [#FORUM] `Placeholder Considerations: Resources, Governance, and Legitimacy in NU4 <https://forum.zcashcommunity.com/t/placeholder-considerations-resources-governance-and-legitimacy-in-nu4/34045>`_
+.. [#ZFSTATEMENT] `Zcash Foundation Guidance on Dev Fund Proposals <https://www.zfnd.org/blog/dev-fund-guidance-and-timeline/>`_
+.. [#ZIPGUIDE] `ZIP Guide <https://github.com/zcash/zips/blob/master/zip-0000.rst>`_
```

Comment:
This is the wrong URL.

```suggestion
.. [#ZIPGUIDE] `ZIP Guide <https://zips.z.cash/zip-guide>`_
```

Reply:
Nitpick: reference tags are usually lowercase except where they're referring to acronyms (the latter rule isn't followed entirely consistently in existing ZIPs). So to follow the style used in other ZIPs, these would be:
- RFC2119
- NU-pipeline
- about-ZF
- forum
- ZF-statement
- zip-guide

(Yeah I know, ZIP is an acronym. I may get around to changing that in other ZIPs to make it more consistent, but probably as a batch change.)